packages/alfa-wcag/test/sia-r11/rule.spec.tsx

```diff
@@ -7,23 +8,36 @@ import { SIA_R11 } from "../../src/sia-r11/rule";
 import { documentFromNodes } from "../helpers/document-from-nodes";
 import { outcome } from "../helpers/outcome";
 
+const device = {
+  type: DeviceType.Screen,
+  viewport: {
+    width: 1920,
+    height: 1080,
+    orientation: Orientation.Landscape
+  },
+  display: {
+    resolution: 1,
+    scan: Scan.Progressive
+  }
+};
+
```

Comment:
`getDefaultDevice()` was made for this purpose: https://github.com/Siteimprove/alfa/blob/master/packages/alfa-device/src/get-default-device.ts